src/phasorpy/cluster.py

```diff
@@ -0,0 +1,193 @@
+"""Cluster phasor coordinates.
+
+The `phasorpy.cluster` module provides functions to:
+
+- cluster phasor coordinates using Gaussian Mixture Model (GMM):
+    - :py:func:`phasor_cluster_gmm`
+
+"""
+
+from __future__ import annotations
+
+__all__ = ['phasor_cluster_gmm']
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import Any, ArrayLike, NDArray
+
+import math
+from collections.abc import Sequence
+
+import numpy
+from sklearn.mixture import GaussianMixture
+
+from ._utils import parse_skip_axis
+
+
+def phasor_cluster_gmm(
+    real: NDArray[numpy.float64],
+    imag: NDArray[numpy.float64],
+    sigma: float = 2.0,
+    /,
+    *,
+    clusters: int = 1,
+    skip_axis: int | Sequence[int] | None = None,
+    **kwargs: Any,
+) -> tuple[
+    tuple[float, ...],
+    tuple[float, ...],
+    tuple[float, ...],
+    tuple[float, ...],
+    tuple[float, ...],
+]:
+    """Return elliptic clusters in phasor coordinates using GMM.
+
+    Fit a Gaussian Mixture Model (GMM) to the provided phasor coordinates and
+    extract the parameters of ellipses that represent each cluster according
+    to [1]_.
+
+    Parameters
+    ----------
+    real : array_like
+        Real component of phasor coordinates.
+    imag : array_like
+        Imaginary component of phasor coordinates.
+    sigma: float, default = 2.0
+        Scaling factor for the radii of major and minor axes. By default, it is set
+        to 2, which corresponds to the scaling of eigenvalues for a 95 per cent confidence
+        ellipse.
+    clusters : int, optional
+        Number of Gaussian distributions to fit to phasor coordinates.
+        Defaults to 1.
+    skip_axis : int or sequence of int, optional
+        Axes to skip in the data (useful for multi-dimensional arrays).
+    **kwargs
+        Additional keyword arguments passed to
+        :py:class:`sklearn.mixture.GaussianMixture`.
+
+        Common options include:
+        - covariance_type : {'full', 'tied', 'diag', 'spherical'}
+        - max_iter : int, maximum number of EM iterations
+        - random_state : int, for reproducible results
```

Comment:
Add an empty line before the list.

```suggestion
        Common options include:

        - covariance_type : {'full', 'tied', 'diag', 'spherical'}
        - max_iter : int, maximum number of EM iterations
        - random_state : int, for reproducible results
```